assets/javascripts/discourse/templates/admin-wizards.hbs

```diff
@@ -1,14 +1,40 @@
 {{#admin-nav}}
   {{nav-item route="adminWizardsWizard" label="admin.wizard.nav_label"}}
-  {{nav-item route="adminWizardsCustomFields" label="admin.wizard.custom_field.nav_label"}}
-  {{nav-item route="adminWizardsSubmissions" label="admin.wizard.submissions.nav_label"}}
+  {{nav-item
+    route="adminWizardsCustomFields"
+    label="admin.wizard.custom_field.nav_label"
+  }}
+  {{nav-item
+    route="adminWizardsSubmissions"
+    label="admin.wizard.submissions.nav_label"
+  }}
   {{#if siteSettings.wizard_apis_enabled}}
     {{nav-item route="adminWizardsApi" label="admin.wizard.api.nav_label"}}
   {{/if}}
   {{nav-item route="adminWizardsLogs" label="admin.wizard.log.nav_label"}}
-  {{nav-item route="adminWizardsManager" label="admin.wizard.manager.nav_label"}}
+  {{nav-item route="adminWizardsManager" label="admin.wizard.manager.nav_label"
+  }}
+
+  <div class="admin-actions">
+    <a
+      target="_blank"
+      class="btn btn-pavilion-pro"
+      rel="noreferrer noopener"
+      href={{siteSettings.wizard_pro_support_url}}
```

Comment:
@keegangeorge We'll have to hardcode this actually. Remember this plugin goes on the client's infrastructure. Let's set it to https://thepavilion.io/w/support.